### ISSUE TITLE
Feature Request: E-Mail notification

### ISSUE THREAD

**Issue by mcnesium** [nextcloud/spreed#944]
I would like to see a mail notification for every user in a chat group

* upon group chat creation
* when there is activity

if a user is not logged in and will not see the regular nextcloud notification

See e.g. [mattermost behavior](https://docs.mattermost.com/help/getting-started/configuring-notifications.html#email-notifications)

**Comment by mario** [comment]
There's a push notification for phones though?

**Comment by florianwgnr** [comment]
When using the desktop sync client or nextcloud in your browser, you get notifications on invitations to a call.

Also there is some kind of option to send an email when receiving an invitation under "settings/user/additional", you should try this.

Cheers

**Comment by mcnesium** [comment]
All of this is true, still when people do not regularly use nextcloud, they neither have phone apps nor desktop clients installed on their devices. They use Whatsapp and Slack and stuff and don't even care about what's the matter with that. If you want them to use decentralized and privacy-aware infrastructure, you need to have them get notified in a way that is convenient, for them as well as for you.

So with a mail notification system in analogy to those "Alice has shared this file with you" mails, you add people to a chat group and just start talking, and they get mails with the latest content of the discussion along with a "join the discussion"-button, that opens up the chat room in the browser. Maybe there could even be a "reply by mail" feature, but this might be a bit too much to ask here now ;)

**Comment by nickvergessen** [comment]
We are not going to send emails instead of notifications.

If people dont want to install a client, they will have to wait for the time when someone writes an app which sends Emails instead of push notifications.

**Comment by nickvergessen** [comment]
> Noone asked to send e-mails instead of notifications.

I meant "for" 

> I understand that this may not be your top priority, and I may be wrong with my assumption, the file-share-mail-notification module can easily adopted for this. Still, maybe sometime in the future, some contributor comes along and PRs this.

Well if we (nextcloud) want to fix the issue you are describing, it should be by sending an email for each notification, not only the ones for Talk.
That is why I closed this here, because it should be done in general and not specific for Talk.

> But you went ahead and closed this issue right away and your answer sounds like you don't actually want a larger number of people to use Nextcloud Talk, I guess. And thats pretty much disappointing.

You are over-interpreting here :wink:

**Comment by mcnesium** [comment]
ok I just looked at this *Additional settings* panel in the settings for the first time in years. I found the *You were invited to a talk room or had a call* setting, that can be checked for mails as well as stream notification. This made me realize how notifications work: any app just sends an event to this thing, and from there it is decided how the user gets notified. That is cool and actually makes sense, so I am sorry for misphrasing my feature request.

So lets try again and put it this way: the Talk app should add some more settings to the notifications system:
* *There has been activity in a talk room after your last login*
* *You were @-mentioned in a talk room*
* *You have been removed from a talk room*
* *You have been promoted to moderator*
* *You have been demoted*
* …

What about that? :)
